packages/zcli-apps/src/utils/createApp.ts

```diff
@@ -41,20 +41,20 @@ export const deployApp = async (method: string, url: string, upload_id: number,
     installationPayload = { upload_id }
   }
   const installationOptions = {
-    body: JSON.stringify(installationPayload),
-    method,
+    data: JSON.stringify(installationPayload),
+    method: method,
```

Comment:
any reason for being explicit here?

Reply:
Fixed in cd0973df173afd6322f338021f3825bb0ff97efb